pkg/e2e/autoscaler/autoscaler.go

```diff
@@ -215,62 +216,89 @@ var _ = g.Describe("[Feature:Machines] Autoscaler should", func() {
 		g.By("Getint target machineSet")
 		machinesets, err := e2e.GetMachineSets(context.TODO(), client)
 		o.Expect(err).NotTo(o.HaveOccurred())
-		o.Expect(len(machinesets)).To(o.BeNumerically(">", 0))
+		o.Expect(len(machinesets)).To(o.BeNumerically(">", 1))
 
-		targetMachineSet := machinesets[0]
-		glog.Infof("Target machineSet %s", targetMachineSet.Name)
+		targetMachineSet0 := machinesets[0]
+		glog.Infof("Target machineSet0 %s", targetMachineSet0.Name)
+		targetMachineSet1 := machinesets[1]
+		glog.Infof("Target machineSet1 %s", targetMachineSet1.Name)
 
 		// When we add support for machineDeployments on the installer, cluster-autoscaler and cluster-autoscaler-operator
 		// we need to test against deployments instead so we skip this test.
-		if ownerReferences := targetMachineSet.GetOwnerReferences(); len(ownerReferences) > 0 {
+		if ownerReferences0 := targetMachineSet0.GetOwnerReferences(); len(ownerReferences0) > 0 {
 			// glog.Infof("MachineSet %s is owned by a machineDeployment. Please run tests against machineDeployment instead", targetMachineSet.Name)
-			g.Skip(fmt.Sprintf("MachineSet %s is owned by a machineDeployment. Please run tests against machineDeployment instead", targetMachineSet.Name))
+			g.Skip(fmt.Sprintf("MachineSet %s is owned by a machineDeployment. Please run tests against machineDeployment instead", targetMachineSet0.Name))
+		}
+
+		if ownerReferences1 := targetMachineSet1.GetOwnerReferences(); len(ownerReferences1) > 0 {
+			// glog.Infof("MachineSet %s is owned by a machineDeployment. Please run tests against machineDeployment instead", targetMachineSet.Name)
+			g.Skip(fmt.Sprintf("MachineSet %s is owned by a machineDeployment. Please run tests against machineDeployment instead", targetMachineSet1.Name))
 		}
 
 		g.By("Create ClusterAutoscaler object")
 		clusterAutoscaler := clusterAutoscalerResource()
 		err = client.Create(context.TODO(), clusterAutoscaler)
 		o.Expect(err).NotTo(o.HaveOccurred())
 
-		initialNumberOfReplicas := pointer.Int32PtrDerefOr(targetMachineSet.Spec.Replicas, e2e.DefaultMachineSetReplicas)
+		initialNumberOfReplicas0 := pointer.Int32PtrDerefOr(targetMachineSet0.Spec.Replicas, e2e.DefaultMachineSetReplicas)
+		initialNumberOfReplicas1 := pointer.Int32PtrDerefOr(targetMachineSet1.Spec.Replicas, e2e.DefaultMachineSetReplicas)
+		glog.Infof("initialNumberOfReplicas0 %v, initialNumberOfReplicas1 %v", initialNumberOfReplicas0, initialNumberOfReplicas1)
```

Comment:
nit: can we log more user friendly? also should use %d e.g `Infof(Initial number of replicas for %q is %d, targetMachineSet0.Name, initialNumberOfReplicas0)`